spec/api-web-contents-view-spec.ts

```diff
@@ -17,6 +17,20 @@ describe('WebContentsView', () => {
     new WebContentsView({});
   });
 
+  it('accepts existing webContents object', async () => {
+    const currentWebContentsCount = webContents.getAllWebContents().length;
+
+    const wc = (webContents as typeof ElectronInternal.WebContents).create({ sandbox: true });
+    await wc.loadURL('about:blank');
+
+    const webContentsView = new WebContentsView({
+      webContents: wc
+    });
+
+    expect(webContentsView.webContents).to.eq(wc);
+    expect(webContents.getAllWebContents().length).to.equal(currentWebContentsCount + 1, 'expected only single webcontents to be created');
```

Comment:
this test should destroy the webContents after it finishes (regardless of success or failure). I recommend the [`defer()`](https://github.com/electron/electron/blob/9b0409f7c91b1dd8e1554f8428972c1b0f171766/spec/lib/spec-helpers.ts#L35) helper.